### ISSUE TITLE
Escaping issue?

### ISSUE THREAD

**Issue by renefritze** [atextor/turtle-formatter#3]
I want to format a Turtle file that looks something like this:
```
@prefix dc:      <http://purl.org/spar/datacite/> .
@prefix doi:     <https://doi.org/> .
something a :publication ; rdfs:label "Paper title" ;
    dc:hasIdentifier doi:10.1137\/1.9781611970937 . 
```

The forward slash escaping is necessary for us to pass validation with `riot` from the Jena package. Inputting above text into
`owl write` I get (Where line/col matches the slash position)
```
Error: Encountered " "\\" "\\ "" at line 32, column 30.
Was expecting one of:
    ";" ...
    "," ...
    "." ...
```

Is this an issue with the input parser? Is `riot` wrong in validating this?

**Comment by atextor** [comment]
Hi, the good news is: I can reproduce the problem with the following model:
```turtle
@prefix dc:      <http://purl.org/spar/datacite/> .
@prefix doi:     <https://doi.org/> .
@prefix : <http://example.org#> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .

:something a :publication ;
    rdfs:label "Paper title" ;
    dc:hasIdentifier doi:10.1137\/1.9781611970937 .
```

turtle-formatter and owl-cli also use Jena as an RDF library, which provides [multiple ways](https://jena.apache.org/documentation/io/rdf-input.html) to parse a file. As it did not use the same method as `riot`, I built a minimal project that tries to load the file using both ways. Both ways load the file as intended, so this is not an issue with either Jena's libraries or the `riot` tool. I'm not sure yet what the problem is, but it's most certainly a bug.

**Comment by atextor** [comment]
After some investigation, it seems this is caused by Jena's internal RDF parsers handling escapes differently. The RIOT parser which handles this correctly is added automatically via service loading when `jena-arq` is added as a dependency. For turtle-formatter, this is the case, while for owl-cli it isn't, so this is actually a an issue there: atextor/owl-cli#1.

**Comment by atextor** [comment]
@renefritze The fix for the broken parsing is done. Furthermore, serialization of escapes was broken as well (see #4). Both fixes are in the just released [owl-cli version 1.2.0](https://github.com/atextor/owl-cli/releases/tag/v1.2.0).

**Comment by renefritze** [comment]
Excellent, thank you.
